content/20.treatments.md

```diff
@@ -188,13 +188,121 @@ Summarize the state of the antiviral approach.
+
+###### Summary
+
+However, the overall lack of human studies in this area means there is limited evidence as to whether these nutraceuticals could affect COVID-19 infection.  
+
+##### Zinc Supplements
+
+There is evidence that nutrient supplements may exhibit some benefit against RNA viral infections.
+Zinc is a trace metal obtained from dietary sources or supplementation that is important for the maintenance of immune cells involved in adaptive and innate immunity [@doi:10.1016/j.abb.2016.03.022].
+Zinc supplements can be administered orally as a tablet or as a lozenge and they are available in many forms such zinc picolinate, zinc acetate, and zinc citrate.
+Zinc is also available from dietary sources including meat, seafood, nuts, seeds, legumes, and dairy.
+
+###### Potential Mechanisms
+
+The role of zinc in immune function has been extensively reviewed [@doi:10.1016/j.abb.2016.03.022].
+Zinc is an important signaling molecule and zinc levels can alter host defense systems. 
+In inflammatory situations such as an infection, zinc can regulate leukocyte immune responses and it can activate the nuclear factor kappa-light-chain-enhancer of activated B cells (NF-kB), thus altering cytokine production [@doi:10.4049/jimmunol.179.6.4180; @doi:10.1067/mlc.2001.118108]. 
+In particular, zinc supplementation can increase natural killer cell levels, which are important cells for host defense against viral infections [@doi:10.1126/science.1198687; @doi:10.1016/j.abb.2016.03.022].
+
+###### Current Evidence
+
+Adequate zinc intake has been associated with reduced incidence of infection [@doi:10.1093/ajcn/85.3.837] and antiviral immunity [@doi:10.1093/advances/nmz013].
+Conversely, zinc deficiency is associated with increased susceptibility to infection in the elderly, which can be prevented when zinc supplements are administered [@doi:10.1093/ajcn/85.3.837].
+Clinical trial data supports the utility of zinc to diminish the duration and severity of symptoms associated with common colds when it is provided within 24 hours of the onset of symptoms [@doi:10.1331/1544-3191.44.5.594.Hulisz; @doi:10.2174/1874306401105010051].
+In coronaviruses specifically, in vitro evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a flourescently tagged SARS-CoV) and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015].
+
+###### Summary
+
+While overall there is encouraging data for zinc supplementation against the common cold and viral infections, there is currently limited evidence to suggest zinc supplementation has any beneficial effects against the current novel COVID-19, thus further research is imperative.
+However, it would be advisable to maintain a healthy diet to ensure an adequate zinc status to prevent the likelyhood of an infection. 
```

Comment:
```suggestion
However, it would be advisable to maintain a healthy diet to ensure an adequate zinc status to prevent the likelihood of an infection. 
```